### PR TITLE
Detect review requests on inactive reviewers

### LINKED ISSUE #1382: Cancel review requests on inactive reviewers
If canceling them is hard, we can also just needinfo the triage owner on the bugs associated to the revisions with inactive reviewers.

### PR DESCRIPTION
**Pull request by suhaibmujahid**
Resolves #1382

## Autonag wiki page
```wiki
{{AutonagRule
  | Rule name = Bugs with inactive patch reviewers
  | Purpose   = Unblock bugs by identifying patches that are waiting for review from inactive or unavailable reviewers
  | Action    = Needinfo patch authors to find active reviewers
  | Source    = inactive_reviewer.py 
}}
```

## Dry-run
See https://github.com/mozilla/relman-auto-nag/pull/1531#issuecomment-1184712892

## Checklist

- [x] Type annotations added to new functions
- [x] Docs added to functions touched in main classes
- [x] Dry-run produced the expected results
- [x] The [`to-be-announced`](https://github.com/mozilla/relman-auto-nag/labels/to-be-announced) tag added if this is worth announcing

### CODE REVIEW

File: auto_nag/user_activity.py
Comment (marco-c) [inline auto_nag/user_activity.py:222]: Nit: why the change from elif to if?

File: auto_nag/user_activity.py
Comment (marco-c) [inline auto_nag/user_activity.py:18]:
```suggestion
# The chunk size here should not be more than 100; which is the maximum number of
```

File: templates/inactive_reviewer_needinfo.txt
Comment (marco-c) [inline templates/inactive_reviewer_needinfo.txt:17]:
```suggestion
:{{ nicknames }}, could you please find another reviewer or abandon the patch if it is no longer relevant?
```

Could we add "or abandon the patch if it is no longer relevant" in case the patch is old?

**Comment by marco-c** [comment]
In [D51425](https://phabricator.services.mozilla.com/D51425), we would be warning about `valentin` and `nhnt11`.
`valentin` accepted a prior diff but not the last one, so I guess it makes sense to warn about him (though there are cases where a patch is updated with a typo and so doesn't need another review, but if the revision is still open it means it was more than just a typo otherwise it would have been landed)
`nhnt11` resigned, so either we don't warn about them, or we add a note that specifies that they resigned.

File: templates/inactive_reviewer_needinfo.txt
Comment (marco-c) [inline templates/inactive_reviewer_needinfo.txt:8]: At least for the "unavailable" case, we should give more information and tell the author when the reviewer will be available again (so they know how long it would take if they decided to wait for the reviewer).

File: auto_nag/user_activity.py
Comment (suhaibmujahid) [reply auto_nag/user_activity.py:222]: We have `return` for each statement, so the else is useless. I did it to satisfy Pylint.

File: auto_nag/scripts/inactive_reviewer.py
Comment (suhaibmujahid) [reply auto_nag/scripts/inactive_reviewer.py:107]: We are dropping the whole revision using `continue`:

https://github.com/mozilla/relman-auto-nag/blob/2381415ed42035a82a919f43e9ff894f8988b37e/auto_nag/scripts/inactive_reviewer.py#L113-L123

If you mean not including these reviewers in the first place, we need to know the full list of reviewers before deciding on a revision.

Now we are filtering the reviewers in one place only:

https://github.com/mozilla/relman-auto-nag/blob/2381415ed42035a82a919f43e9ff894f8988b37e/auto_nag/scripts/inactive_reviewer.py#L145-L152

Comment (marco-c) [reply auto_nag/scripts/inactive_reviewer.py:107]: I was thinking of removing e.g. the "group" reviewers from the reviewer list here, so later we don't need to filter them out.

(not important, you can also keep it as it is)

Comment (suhaibmujahid) [reply auto_nag/scripts/inactive_reviewer.py:107]: I keep them because we consider group reviewers as active reviewers; thus, if there are no blocking reviewers who are inactive, we do not warn about the patch (we do not warn if we have at least one active reviewer unless the inactive reviewer is blocking). If I remove group reviewers from the initial list, we cannot tell if the revision has an active reviewer.

File: templates/inactive_reviewer_needinfo.txt
Comment (suhaibmujahid) [reply templates/inactive_reviewer_needinfo.txt:8]: Done in f6f5a5943c3586ccd5d52c75848d31e8dd041d10

File: templates/inactive_reviewer_needinfo.txt
Comment (suhaibmujahid) [reply templates/inactive_reviewer_needinfo.txt]: Done in 201c2f3eb24e6c0b27fc3d7565ab5bb9afb6b8bb

**Comment by suhaibmujahid** [comment]
> In [D51425](https://phabricator.services.mozilla.com/D51425), we would be warning about `valentin` and `nhnt11`. `valentin` accepted a prior diff but not the last one, so I guess it makes sense to warn about him (though there are cases where a patch is updated with a typo and so doesn't need another review, but if the revision is still open it means it was more than just a typo otherwise it would have been landed) `nhnt11` resigned, so either we don't warn about them, or we add a note that specifies that they resigned.

This specific case was not caught because Fallen was shown as available based on phab API, even though you see him `Disabled` in the web interface. This was fixed in 061f1665090254ef1efc11d35ff7a4ab5f099f36 by checking if the user has a role called `disabled`. Also, now we treat resigned reviewers as inactive (we show a note to specify that they resigned).

File: templates/inactive_reviewer_needinfo.txt
Comment (marco-c) [inline templates/inactive_reviewer_needinfo.txt:17]: The "or" should be after the if, right?

**Comment by marco-c** [comment]
Could you upload the dry run results with the latest version of the patch?

File: templates/inactive_reviewer_needinfo.txt
Comment (suhaibmujahid) [reply templates/inactive_reviewer_needinfo.txt]: Yes, nice catch. Fixed in 2eb08c57081f8aa327c4d5e6a31e27ef26be7e21.

**Comment by suhaibmujahid** [comment]
## Dry-run

```
2022-07-14 13:17:45,689 - INFO - Run tool inactive_reviewer.py
2022-07-14 13:19:18,508 - INFO - Tool inactive_reviewer.py has finished.
```

### Email

<details>
<summary>Click to expand the dry-run output!</summary><br>

<p>Hi there,</p>

<p>The following bugs have patches with inactive reviewers:
<table style="border:1px solid black;border-collapse:collapse;" border="1">
<thead>
<tr>
<th>Bug</th><th>Summary</th><th>Patches</th>
</tr>
</thead>
<tbody>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1771550">1771550</a>
</td>
<td>
PiP Subtitles Support for Khan Academy
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D148668">D148668</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1771417">1771417</a>
</td>
<td>
Calculate video decoding fps and store it to Telemetry
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D147539">D147539</a></li>
<li><a href="https://phabricator.services.mozilla.com/D147540">D147540</a></li>
<li><a href="https://phabricator.services.mozilla.com/D147541">D147541</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1761282">1761282</a>
</td>
<td>
Update sad/happy smiley faces to emoji
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D143789">D143789</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1759222">1759222</a>
</td>
<td>
Intermittent toolkit/content/tests/browser/browser_findbar.js | Test timed out -
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D141070">D141070</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1753601">1753601</a>
</td>
<td>
[wayland] Web Extension popup content is trimmed with fractional scaling
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D143033">D143033</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1751990">1751990</a>
</td>
<td>
Remove usages of &#34;six&#34; from &#34;toolkit&#34;
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D137058">D137058</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1739381">1739381</a>
</td>
<td>
Add profiler marker for each AudioNode with their type, and AudioWorkletProcessor.process with the Processor name
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D145167">D145167</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1738755">1738755</a>
</td>
<td>
Accumulate &#34;LABELS_MEDIA_SNIFFER_MP4_BRAND_PATTERN&#34; only for major brand.
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D130237">D130237</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1737889">1737889</a>
</td>
<td>
Generalize stacking context display list retaining code and reuse display items for leaf block frames
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D143397">D143397</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1709199">1709199</a>
</td>
<td>
Shortcut modal - change to single column layout
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D133728">D133728</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1707731">1707731</a>
</td>
<td>
browser/components/preferences/tests/browser_searchQuickSuggest.js test fails on apple silicon
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D113525">D113525</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1702761">1702761</a>
</td>
<td>
Define and implement core APIs for Zabkainstrumentation
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D110681">D110681</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1659972">1659972</a>
</td>
<td>
Start documenting SpiderMonkey Conventions
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D87585">D87585</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1649427">1649427</a>
</td>
<td>
Add DumpCustomAttributes function for console debugging checkers
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D81698">D81698</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1624907">1624907</a>
</td>
<td>
Scope tree.css styles to tree custom element
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D71652">D71652</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1624546">1624546</a>
</td>
<td>
Include vcruntime140_1.dll for VS2019 16.5 x64 from the redist directory in the package
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D68010">D68010</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1624170">1624170</a>
</td>
<td>
history.pushState() is allowed to add as many entries as it wants, as fast as it wants
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D67775">D67775</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1623856">1623856</a>
</td>
<td>
Shift-clicking the refresh button does not duplicate page in a new window
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D86464">D86464</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1618557">1618557</a>
</td>
<td>
It seems that LoadInfo&#39;s top-level document constructor sets up the wrong top-level anti-tracking principal
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D79098">D79098</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1605232">1605232</a>
</td>
<td>
Improve workflow when running taskcluster jobs locally
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D57876">D57876</a></li>
<li><a href="https://phabricator.services.mozilla.com/D57881">D57881</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1602305">1602305</a>
</td>
<td>
Caret is not visible in textbox (Firefox RTL)
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D58148">D58148</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1599612">1599612</a>
</td>
<td>
Remove methods that only return NS_OK
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D54824">D54824</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1588067">1588067</a>
</td>
<td>
Turn on the prefs to enable the recognition part of Web Speech API in Nightly desktop
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D50698">D50698</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1587413">1587413</a>
</td>
<td>
Intermittent TEST-UNEXPECTED-PASS | /fetch/range/sw.https.window.html | Ranged response not allowed following no-cors ranged request - expected FAIL
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D53114">D53114</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1587396">1587396</a>
</td>
<td>
Thunderbird-only read of AppType from root docshell in CanLoadImage uses nsIDocShellTreeItem
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D67173">D67173</a></li>
<li><a href="https://phabricator.services.mozilla.com/D67174">D67174</a></li>
<li><a href="https://phabricator.services.mozilla.com/D67175">D67175</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1580983">1580983</a>
</td>
<td>
Investigate using &lt;toolbox&gt; as an HTML element
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D51168">D51168</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1579075">1579075</a>
</td>
<td>
:lang pseudo selector doesn&#39;t propagate across shadow boundaries.
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D45638">D45638</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1570703">1570703</a>
</td>
<td>
Watch Expressions should always show up-to-date values.
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D43826">D43826</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1563221">1563221</a>
</td>
<td>
Remove older unused unix from NSS source code.
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D36755">D36755</a></li>
<li><a href="https://phabricator.services.mozilla.com/D36756">D36756</a></li>
<li><a href="https://phabricator.services.mozilla.com/D36757">D36757</a></li>
<li><a href="https://phabricator.services.mozilla.com/D37215">D37215</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1562317">1562317</a>
</td>
<td>
stack fixing scripts do nothing if function name is known
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D36383">D36383</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1559791">1559791</a>
</td>
<td>
browser.find.find(&#34;&#34;, { rectList: true }) returns incomplete results
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D39550">D39550</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1559226">1559226</a>
</td>
<td>
MOZ_STACK_CLASS fails to detect heap-allocation of annotated classes that live in nsTArray/AutoTArray members of other heap-allocated structures
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D37682">D37682</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1554721">1554721</a>
</td>
<td>
IonMonkey never execute optimized divisions by a constant divisor.
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D32738">D32738</a></li>
<li><a href="https://phabricator.services.mozilla.com/D32739">D32739</a></li>
<li><a href="https://phabricator.services.mozilla.com/D32740">D32740</a></li>
<li><a href="https://phabricator.services.mozilla.com/D32741">D32741</a></li>
<li><a href="https://phabricator.services.mozilla.com/D32742">D32742</a></li>
<li><a href="https://phabricator.services.mozilla.com/D32743">D32743</a></li>
<li><a href="https://phabricator.services.mozilla.com/D32744">D32744</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1544906">1544906</a>
</td>
<td>
Get Element Property should JSON serialise its response value
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D27918">D27918</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1533871">1533871</a>
</td>
<td>
Enable a more intuitive way to create transforms specific to tasks in a given .yml file
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D23418">D23418</a></li>
<li><a href="https://phabricator.services.mozilla.com/D46656">D46656</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1503203">1503203</a>
</td>
<td>
Add the three row view when docked to the side
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D20812">D20812</a></li>
<li><a href="https://phabricator.services.mozilla.com/D28812">D28812</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1503084">1503084</a>
</td>
<td>
Intermittent accessible/tests/browser/e10s/browser_caching_name.js | &lt;random text&gt; - Got name from (alt|value), expected test4
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D46700">D46700</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1472169">1472169</a>
</td>
<td>
&lt;textarea&gt; validity state is not preserved correctly after a clone.
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D1878">D1878</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1465995">1465995</a>
</td>
<td>
Implement ShowPageAction for declarativeContent API
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D25591">D25591</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1465994">1465994</a>
</td>
<td>
Implement setIcon as declarativeContent API
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D42313">D42313</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1455425">1455425</a>
</td>
<td>
expose TRR (DNS over HTTPS) configuration in browser settings
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D51425">D51425</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1446819">1446819</a>
</td>
<td>
Crash [@ mozalloc_abort] with UNIMPLEMENTED abort message and Debugger
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D21040">D21040</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1437051">1437051</a>
</td>
<td>
Intermittent browser/components/preferences/in-content/tests/browser_extension_controlled.js | This test exceeded the timeout threshold. It should be rewritten or split up. If that&#39;s not possible, use requestLongerTimeout(N), but only as a last resort. -
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D3453">D3453</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1434134">1434134</a>
</td>
<td>
Remove a few PulseAudio-specific sandboxing rules missed in bug 1386019
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D114360">D114360</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1406475">1406475</a>
</td>
<td>
Remove Preferences.jsm usage from Sync
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D11350">D11350</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1399146">1399146</a>
</td>
<td>
Unbreak jprof and minor cleanup
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D35853">D35853</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1354578">1354578</a>
</td>
<td>
Replace legacy actions with WebDriver action commands in Marionette client
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D15931">D15931</a></li>
</ul>
</td>
</tr>
<tr >
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1152884">1152884</a>
</td>
<td>
Add timeline markers to trace synchronous XHR
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D23800">D23800</a></li>
</ul>
</td>
</tr>
<tr bgcolor="#E0E0E0">
<td >
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=301888">301888</a>
</td>
<td>
Bookmarks cut instead of opened in new tab from Bookmarks Toolbar Folder (shortcut key &#34;t&#34; in the right-click context menu is inconsistent with that of links)
</td>
<td>
<ul>
<li><a href="https://phabricator.services.mozilla.com/D44078">D44078</a></li>
</ul>
</td>
</tr>
</tbody>
</table>
</p>
<p>Sincerely,<br>
Autonag bot
</p>
<p>
See the search query on <a href="https://mzl.la/3AObrUq">Bugzilla</a>.
</p>

<p>
See the source / report issues on the <a href="https://github.com/mozilla/relman-auto-nag/">GitHub repo</a>
</p>

</details>


### Needinfo comment examples

<details>
<summary>Click to expand the dry-run output! (One patch)</summary><br>

The following patch is waiting for review from an inactive reviewer:

| ID  | Title |  Author | Reviewer Status |
| --- | :---- |  ------ | --------------- |
| [D114360](https://phabricator.services.mozilla.com/D114360) |  Bug 1434134 - Remove a few PulseAudio-specific sandboxing rules missed in bug 1386019 r?jld | [gerard-majax](https://bugzilla.mozilla.org/user_profile?user_id=41385)  |[jld](https://phabricator.services.mozilla.com/p/jld/): Resigned |


:gerard-majax, could you please find another reviewer or abandon the patch if it is no longer relevant?

For more information, please visit [auto_nag documentation](https://wiki.mozilla.org/Release_Management/autonag#inactive_reviewer.py).

</details>


<details>
<summary>Click to expand the dry-run output! (Multiple patches)</summary><br>

The following patches are waiting for review from an inactive reviewer:

| ID  | Title |  Author | Reviewer Status |
| --- | :---- |  ------ | --------------- |
| [D57876](https://phabricator.services.mozilla.com/D57876) |  Bug 1605232 - Add run-task --keep option to allow rerunning tasks locally without re-downloading compilers | [sfink](https://bugzilla.mozilla.org/user_profile?user_id=359004)  |[tomprince](https://phabricator.services.mozilla.com/p/tomprince/): Inactive |
| [D57881](https://phabricator.services.mozilla.com/D57881) |  Bug 1605232 - Add run-task --existing-gecko-checkout option for running taskcluster job locally using host directory mounted into container | [sfink](https://bugzilla.mozilla.org/user_profile?user_id=359004)  |[tomprince](https://phabricator.services.mozilla.com/p/tomprince/): Inactive |


:sfink, could you please find another reviewer or abandon the patch if it is no longer relevant?

For more information, please visit [auto_nag documentation](https://wiki.mozilla.org/Release_Management/autonag#inactive_reviewer.py).

</details>



<details>
<summary>Click to expand the dry-run output! ( One patch and multiple inactive reviewers)</summary><br>

The following patch is waiting for review from an inactive reviewer:

| ID  | Title |  Author | Reviewer Status |
| --- | :---- |  ------ | --------------- |
| [D51425](https://phabricator.services.mozilla.com/D51425) |  Bug 1455425 - expose TRR (DNS over HTTPS) configuration in browser settings, r?mixedpuppy | [baku](https://bugzilla.mozilla.org/user_profile?user_id=446257)  |[valentin](https://phabricator.services.mozilla.com/p/valentin/): Back Jul 31, 2022 |
| | | |[Fallen](https://phabricator.services.mozilla.com/p/Fallen/): Disabled |
| | | |[nhnt11](https://phabricator.services.mozilla.com/p/nhnt11/): Resigned |


:baku, could you please find another reviewer or abandon the patch if it is no longer relevant?

For more information, please visit [auto_nag documentation](https://wiki.mozilla.org/Release_Management/autonag#inactive_reviewer.py).


</details>

**Review by marco-c — DISMISSED** [review]

**Comment by sylvestre** [comment]
please change
"Resigned" => "Resigned from review"

**Review by marco-c — APPROVED** [review]

**Pull request merged**
